release-notes/9.0/preview/preview2/api-diff/Microsoft.NETCore.App/9.0-preview2_System.Net.Http.Headers.md

```diff
@@ -0,0 +1,11 @@
+# System.Net.Http.Headers
+
+``` diff
+ namespace System.Net.Http.Headers {
+     public class NameValueHeaderValue : ICloneable {
+-        protected NameValueHeaderValue(NameValueHeaderValue source);
++        protected internal NameValueHeaderValue(NameValueHeaderValue source);
```

Comment:
This isn't a public API change and can be removed from this diff.

Reply:
Thanks. Removing it. I opened an issue to track this request: https://github.com/dotnet/arcade/issues/14579